CMakeLists.txt

```diff
@@ -139,7 +139,17 @@ if (NGRAPH_ONNX_IMPORT_ENABLE)
     option(NGRAPH_ONNXIFI_ENABLE "Enable ONNX Interface for Framework Integration" TRUE)
 endif()
 
-option(NGRAPH_PLAIDML_ENABLE "Enable the PlaidML backend" FALSE)
+find_package(PlaidML CONFIG)
```

Comment:
I don't see FindPlaidML.cmake in the repo. Is that file installed with PlaidML by a different installer?

Reply:
Yes; there's a PlaidML Python wheel, and there may be other distribution mechanisms in the future.  On the PlaidML side, there's currently a hack SDK target that includes a working plaidml-config.cmake, and a PR-in-progress-once-I-fix-the-Windows-build that builds the plaidml-config.cmake into the Python wheel in a place where cmake expects to find it.

TL;DR: To build ngraph with plaidml, you do something like:
```
pip install plaidml
cmake ${NGRAPH_SRC_DIR}
make
```

Much easier than the current "Untar this, set this env var to this, set such-and-such cmake flag, &c" approach.  :-)